### PR TITLE
emoji pipe lol

### PR DESCRIPTION
**Pull request by tenderlove**
For developer happiness!

[this is a joke PR]

### CODE REVIEW

**Comment by adambutler** [comment]
👍

**Comment by simi** [comment]
Why?

**Comment by robinboening** [comment]
Happiness. But at what cost?

**Comment by AJFaraday** [comment]
I whole-heartedly support this change!

File: version.c [hunk 1]
```diff
@@ -128,6 +128,7 @@ ruby_show_version(void)
 #ifdef HAVE_MALLOC_CONF
     if (malloc_conf) printf("malloc_conf=%s\n", malloc_conf);
 #endif
+    printf("Aaron's Ruby Interpreter 😊\n");
```
Comment (mrfoto) [inline version.c]: We need to talk

**Pull request closed without merging**